frontend/website-redesign/src/components/CultureGrid.re

```diff
@@ -0,0 +1,114 @@
+module Styles = {
+  open Css;
+
+  let container =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      media(
+        Theme.MediaQuery.desktop,
+        [flexDirection(`row), justifyContent(`spaceBetween)],
+      ),
+    ]);
+
+  let grid =
+    style([
+      display(`grid),
+      gridTemplateColumns([`rem(21.)]),
+      unsafe("grid-auto-rows", "min-content"),
```

Comment:
hmm, i think you can just do gridAutoRows([`rem( - the row height here - ) ])

Reply:
I wanted the grid row height to scale with the content that was fed in. It didn't make sense to have a hardcoded value since the one on the `Community` page and the `Grants` page is different.

Reply:
makes sense! sounds good